internal/scheduler/jobdb/job.go

```diff
@@ -49,6 +51,8 @@ type Job struct {
 	queuedVersion int32
 	// Scheduling requirements of this job.
 	jobSchedulingInfo *schedulerobjects.JobSchedulingInfo
+	// Scheduling requirements of this job stored in efficient form.
```

Comment:
```suggestion
	// Resource requirements of this job stored in efficient form.
```

Reply:
(technically it's not the whole set of scheduling requirements, just the resources)

Reply:
done